Renegade X Launcher/VersionCheck.cs

```diff
@@ -33,35 +34,35 @@ public static class VersionCheck
 
         static VersionCheck()
         {
-            LauncherVersion = new Version
+            _launcherVersion = new Version
             {
-                Name = "0.77-dev",
-                Number = 077
+                Name = "0.8",
+                Number = 08

```

Comment:
This version number actually needs to be at least as big as the last one (8 < 77); I'll PR a change to set this to 0 though and instead start making branches for each release with the version number